docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+The `balanced` global *mark* GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some child elements are omitted for clarity:
+
+```xml
+<exclusive-start/>                        (STW pause starts)
+
+<allocation-taxation/>                    (memory threshold trigger recorded)
+
+<cycle-start type="global mark phase"/>   (global mark cycle starts)
+
+<gc-start type="global mark phase"/>      (1st GMP STW subincrement starts)
+
+    <mem-info>                            (memory status before operations)
+
+    <remembered-set>
+
+    </mem-info>         
+
+</gc-start> 
+
+<gc-op type="mark increment" />           (STW copy forward operation completed)
+
+<gc-op  type="class unload" />            (optional STW class unload operation completed)
+
+<gc-end>                                  (partial cycle increment ends)
+
+    <mem-info>                            (memory status after operations)
+          
+    <remembered-set>
+
+    </mem-info>
+
+<gc-end>
+
+<concurrent-start type="GMP work packet processing"/> (1st GMP concurrent subincrement starts)
+
+<exclusive-end/>                                      (STW pause ends)
+
+<concurrent-end type="GMP work packet processing"/>   (1st GMP concurrent subincrement ends)
+
+<gc-op type="mark increment"/>                        (copy forward operation runs concurrently)
+
+</concurrent-end type="GMP work packet processing"/>>
+
+...                                       (partial GC cycles run during STW pauses)
+
+<exclusive-start/>                        (STW pause starts)
+
+<gc-start type="global mark phase"/>                  (2nd STW GMP subincrement starts)
+
+...   
+
+<concurrent-start type="GMP work packet processing"/> (2nd concurrent GMP subincrement starts)
+
+...
+
+<exclusive-end/>
+
+...                                       (more partial GC cycles may run)
+
+<concurrent-end type="GMP work packet processing" />  (2nd concurrent GMP subincrement ends)
+...
+
+</concurrent-end>
+
+...                                       (more partial cycles and GMP increments interleave)
+
+<exclusive-start/>                        (STW pause starts)
+
+...
+
+<gc-start type="global mark phase"/>      (final STW GMP subincrement starts.)
+
+...   
+<gc-end type="global mark phase"/>        (final STW GMP subincrement ends. No concurrent subincrement runs)
+
+<cycle-end type="global mark phase"/>     (end of global mark cycle)
+
+<exclusive-end/>                          (STW pause ends)    
+
+<exclusive-start/>                        (STW pause starts)
+
+<cycle-start type="partial gc" />         (partial cycle starts)
+
+...
+
+<gc-op type="sweep" />                    (Sweep operation associated with global mark cycle runs)
+
+...
+
+<cycle-end type="partial gc"/>            (partial GC cycle ends)
+
+<exclusive-end/>                          (STw pause ends)
+
```

Comment:
it would be cool (not necessarily as part of this PR) if some parts of this are collapsible: each GMP increment, and each PGC cycle, so it's easier to observe a big picture of a GMP cycle

Reply:
I'll look at whether we could do this with the current functionality we have access to.